### ISSUE TITLE
Warning about deprecated constants in the log file.

### ISSUE THREAD

**Issue by ColinRobbins** [mitch-dc/volkswagen_we_connect_id#237]
## Version of the custom_component

V.0.2.0

## Installation method (hacs / manual)

HACS

## Configuration

```yaml

Add your configuration here.

```

## Describe the bug

Deprecation error in the log file with HA 2024.1.1

## Debug log

```text

Logger: homeassistant.components.device_tracker
Source: helpers/deprecation.py:205
Integration: Device Tracker (documentation, issues)
First occurred: 10:45:36 (2 occurrences)
Last logged: 10:45:36

SOURCE_TYPE_GPS was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use SourceType.GPS instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration

```

and

```text

Logger: homeassistant.const
Source: helpers/deprecation.py:205
First occurred: 10:45:35 (5 occurrences)
Last logged: 10:45:35

LENGTH_KILOMETERS was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfLength.KILOMETERS instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration
SPEED_KILOMETERS_PER_HOUR was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfSpeed.KILOMETERS_PER_HOUR instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration
TIME_DAYS was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.DAYS instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration
TIME_MINUTES was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.MINUTES instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration
TEMP_CELSIUS was used from volkswagen_we_connect_id, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please report it to the author of the 'volkswagen_we_connect_id' custom integration


```

**Comment by northalpha** [comment]
this seems to be fixed in master via #243 and is not present in `main` anymore, so a new release will incorporate this.

**Comment by ColinRobbins** [comment]
Problem gone away
